supplemental-ui/partials/header-content.hbs

```diff
@@ -1,46 +1,98 @@
 <header class="header">
-    <nav class="navbar">
-        <div class="navbar-brand">
-            <a class="navbar-item" href="{{{or site.url siteRootPath}}}">{{site.title}}</a>
-            {{#if env.SITE_SEARCH_PROVIDER}}
-            <div class="navbar-item search hide-for-print">
-                <div id="search-field" class="field">
-                    <input id="search-input" type="text" placeholder="Search the docs" {{#if page.home}}
-                        autofocus{{/if}}>
-                </div>
-            </div>
-            {{/if}}
-            <button class="navbar-burger" data-target="topbar-nav">
-                <span></span>
-                <span></span>
-                <span></span>
-            </button>
+  <nav class="navbar">
```

Comment:
This seems similar to the changes in the pr on `docs-ui` repo https://github.com/kaskada-ai/docs-ui/pull/3/files#diff-0fa42a4e57b0c2c5a2dd7f680b6045d2f4311ac556ff375fc4cc3dd959e4ed86

Do we need both?

Reply:
yes, this is the same for them the only difference are the urls, i think i will move them to the UI instead of having them duplicated.